.env

```diff
@@ -1,2 +1,3 @@
 PORT=8080
+jwtSecret=maxim_bricks_2020
 DB_URL=mongodb+srv://berlinum:XpYXHlabIFVtD3Qq@bricks-vv3is.mongodb.net/test?retryWrites=true&w=majority
```

Comment:
This file should not be in git.
Now I know your password ;).

Reply:
😣